subsys/net/l2_wifi_if_conn/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_library()
+zephyr_library_sources_ifdef(CONFIG_L2_WIFI_CONNECTIVITY l2_wifi_conn.c)
+
+zephyr_library_include_directories(${ZEPHYR_BASE}/include)
+zephyr_library_include_directories(${ZEPHYR_BASE}/../nrf/include)
+zephyr_library_include_directories(${ZEPHYR_BASE}/../modules/lib/hostap)
+zephyr_library_include_directories(${ZEPHYR_BASE}/subsys/net/conn_mgr)
```

Comment:
A single instance should be enough, it takes a list.

Reply:
Done